source/include/gridrec/filters.h

```diff
@@ -58,12 +58,6 @@
 #endif
 #define ANSI
```

Comment:
These compiler defs and include statements should not be in the public headers because they are unused here. If they are implementation related, they should be in the source files.